echo-mysql/README.md

```diff
@@ -51,36 +37,75 @@ To generate testcases we just need to make some API calls. You can use Postman,
 ```
 
 
-3. Short URL:
+3. Shorten URL:
 ```bash
 -> curl -X POST http://localhost:9090/shorten -H "Content-Type: application/json" -d '{"url": "https://github.com"}'
 ```
 
 
-4. Resolve short code:
+4. Resolve shortened code:
 
 ```bash
 -> curl -X GET http://localhost:9090/resolve/4KepjkTT
 ```
 
 Now both these API calls were captured as a testcase and should be visible on the Keploy CLI. You should be seeing an app named keploy folder with the test cases we just captured and data mocks created.
 
-![alt text](https://github.com/Hermione2408/samples-go/blob/app/echo-mysql/img/keploy_record.png?raw=true)
+![alt text](https://github.com/keploy/samples-go/blob/main/echo-mysql/img/keploy_record.png?raw=true)
 
-# Run the captured testcases
+## Run the captured testcases
 
 Now that we have our testcase captured, run the test file.
 
 ```bash
-sudo -E env PATH=$PATH oss test -c "./echo-mysql" --delay 20
+keploy test -c "docker compose up --no-deps app" --delay 20
```

Comment:
why do we need this flag? can it work without this flag?

Reply:
I removed the flag here. https://github.com/keploy/samples-go/pull/146/commits/f1ad70a7a5ede43bc20d514ed99428a895ffc41c

My intention was to show that keploy test can work even without starting the mysql container. But in practice, we don't need to do this every time.